### ISSUE TITLE
JAWS not reading out if there is a submenu in Navigation

### ISSUE THREAD

**Issue by senthil-kk20** [w3c/aria-practices#2627]
In Menubar navigation, https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/

The level 3 menu items are not read out by JAWS but it's working great with the Mac voice reader. JAWS just fails to see the submenu and never says it is "collapsed". Is there any workaround already that I am not aware of?

**Comment by JAWS-test** [comment]
Maybe you mean this JAWS bug: https://github.com/FreedomScientific/VFO-standards-support/issues/415

**Comment by JAWS-test** [comment]
Workaround would be to refer to the submenu in text form. But that's not great, because then in other screen readers the reference to the submenu occurs twice

**Comment by a11ydoer** [comment]
@senthil-kk20 Thanks for your feedback. Since this is duplicate of other issue and screen reader issue, I will close the issue. as @JAWS-test mentioned, you can follow up this issue via [JAWS bug](https://github.com/FreedomScientific/VFO-standards-support/issues/415) Then it would be a duplicate of [#2248](https://github.com/w3c/aria-practices/issues/2248)
